demisto_sdk/commands/content_graph/tests/create_content_graph_test.py

```diff
@@ -913,3 +914,28 @@ def test_create_content_graph_with_python_version(
         )
         assert expected_python_version == integrations[0].python_version
         assert dockerhub_api_mocker.called == is_taken_from_dockerhub
+
+
+def test_add_relationship__to_Pydentic_and_back():
+    """
+    Given:
+        - A set of relationship data
+    When:
+        - This data is added to a Relationship object using the 'add' method
+    Then:
+        - Verify that the data was added through the Pydantic 'Relationship' object and complies with its specifications:
+            - Convert boolean strings to boolean objects.
+            - Include only properties defined in the Pydantic model.
+            - Exclude properties with a value of None from the object.
+        - Verify that the data was converted back into a dictionary.
+    """
+    relationships = Relationships()
+    relationships.add(
+        RelationshipType.DEPENDS_ON,
+        mandatorily="True",
```

Comment:
please add more relationships tests. For instance check `true` string and invalid `target_type`

Reply:
Done